sycl/test-e2e/syclcompat/launch/launch_policy_neg.cpp

```diff
@@ -0,0 +1,169 @@
+/***************************************************************************
+ *
+ *  Copyright (C) Codeplay Software Ltd.
+ *
+ *  Part of the LLVM Project, under the Apache License v2.0 with LLVM
+ *  Exceptions. See https://llvm.org/LICENSE.txt for license information.
+ *  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCLcompat API
+ *
+ *  launch_policy_neg.cpp
+ *
+ *  Description:
+ *     Negative tests for new launch_policy.
+ **************************************************************************/
+
+// RUN: not %clangxx -std=c++20 -fsycl -fsycl-device-code-split=per_kernel -fsycl-targets=%{sycl_triple} %s -o %t.out 2>&1 | FileCheck -vv %s
```

Comment:
Do we need `-fsycl-device-code-split=per_kernel`?

Reply:
It's in all the other tests as well.

Reply:
Removed where we don't need it :+1: